src/components/Popups/ReplyField/PopupList.tsx

```diff
@@ -0,0 +1,41 @@
+import * as React from 'react';
+import { Modifier as PopperModifier, Options } from '@popperjs/core';
```

Comment:
Let's try to avoid importing directly from popperjs, since it defeats the abstraction of our Popper helper.